### PR TITLE
[GDB-10989] Reorganize Repo

### PR DESCRIPTION
**Pull request by simeonzhekofff**
- Added CI/CD for simple checks and variables update in the README.md
- Added templates for github
- Added security checks via TFSec
- Fixed Security issues
- Added CHANGELOG.md
- Added information about CODEOWNERS and how to contribute
- Added editor config

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  analyze:
+    name: Analyze the Terraform scripts
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
```
Comment (mihailradkov) [inline .github/workflows/ci.yml:19]: Prefer SHAs https://github.com/Ontotext-AD/terraform-azure-graphdb/blob/main/.github/workflows/ci.yml

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        run: terraform init
+
+      - name: Run Terraform format check
+        run: terraform fmt -check -recursive
+
+      - name: Run Terraform validate check
+        run: terraform validate
+
+      - name: Run tfsec
+        uses: aquasecurity/tfsec-sarif-action@v0.1.4
```
Comment (mihailradkov) [inline .github/workflows/ci.yml:34]: Didn't we talk about using trivy?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,90 @@
 # GraphDB deployment package on GCP
 
+This terraform module allows you to deploy GraphDB instance in GCE(Google Compute Engine).
```
Comment (mihailradkov) [inline README.md:3]:
```suggestion
This terraform module allows you to deploy GraphDB instance in Google Compute Engine.
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,90 @@
...
+integration with external search applications, compatibility with industry standards, and both community and commercial
+support, GraphDB is the
+preferred database choice of both small independent developers and big enterprises.
+
+## Features
+
+The module provides the building blocks of configuring, deploying and provisioning a single VM of GraphDB
+using Google Compute Engine. Key features of the module include:
+
+- Google Cloud Engine Compute Instance
```
Comment (mihailradkov) [inline README.md:42]:
```suggestion
- Google Cloud Compute Engine instance
```

File: README.md [hunk 5]
```diff
@@ -1,5 +1,90 @@
...
+- Google Compute Firewall
```
Comment (mihailradkov) [inline README.md:43]:
```suggestion
- Google Cloud Compute Firewall
```

?

File: README.md [hunk 6]
```diff
@@ -33,3 +118,39 @@ cft blueprint metadata -v -p .
...
   - https://github.com/g-awmalik/terraform-google-canonical-mp/blob/main/metadata.display.yaml
 - Blueprint reference https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/bpmetadata
+
+## Local Development
+
+Instead of using the module dependency, you can create a local variables file named `terraform.tfvars` and provide
+configuration overrides there.
+Here's an example of a `terraform.tfvars` file:
+
+### terraform.tfvars
```
Comment (mihailradkov) [inline README.md:128]: Remove this heading

File: .github/workflows/ci.yml (hunk 2 above)
Comment (simeonzhekofff) [reply .github/workflows/ci.yml:34]: I thought that we will update all the repos to trivy in another task, but okay.

File: README.md (hunk 6 above)
Comment (simeonzhekofff) [reply README.md]: Fixed.

File: README.md (hunk 5 above)
Comment (simeonzhekofff) [reply README.md]: Removed.

File: README.md (hunk 4 above)
Comment (simeonzhekofff) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (simeonzhekofff) [reply README.md]: Fixed

File: .github/workflows/ci.yml (hunk 2 above)
Comment (simeonzhekofff) [reply .github/workflows/ci.yml]: Fixed

File: .github/workflows/ci.yml (hunk 1 above)
Comment (simeonzhekofff) [reply .github/workflows/ci.yml]: Fixed

**Review by mihailradkov — APPROVED** [review]

**Pull request merged**